spec/ruby/optional/capi/util_spec.rb

```diff
@@ -293,4 +293,34 @@
     end
   end
 
+  # At least MRI redefines strtod via a macro to its own implementation
```

Comment:
Could you be more explicit here with something like:
`ruby/util.h redefines strtod() as a macro calling ruby_strtod()`?